src/main/resources/templates/reviews.html

```diff
@@ -17,5 +17,16 @@ <h3 th:text="${review.price}" class="review__price">Default Price</h3>
 
 </div>
 
+<form action="/reviews/add-reviews" method="POST">
+    <input type ="text" name="name" placeholder="Review Name" />
+    <button type="submit">Add Review</button>
+</form>
```

Comment:
Review takes more args than this

Reply:
fixed via https://github.com/2020-Spring-Cohort/reviews-site-models/pull/41/commits/4a8682409c75e91bd225174719d1faef4904f7a4#diff-fd2eeff5cc477fbeadd8f39a1f01e363